Database.md

```diff
@@ -87,21 +87,31 @@ erDiagram
 		UUID    archive_uuid    "DEFAULT NULL; FOREIGN KEY archives.uuid"
 		STRING  volume          "DEFAULT NULL"
 		STRING  name            "INDEX; NOT NULL; UNIQUE (owner_id, parent_id, name)"
+		INT     size            ""
+		BOOL    is_shared       "DEFAULT FALSE"
```

Comment:
I think we don´t need this field. We can just `SELECT COUNT` from the `shared_files` to check if the file is shared.

Reply:
If I remember correctly this is to speed up one of the most common use cases which is listing the user's files.

(In the front) In order for the user to quickly know if a file is shared, an icon could be displayed near the file name.

It could be expensive to `SELECT COUNT` for every individual file on the listing.

Reply:
I think the user's files are listed using a different query (`SELECT fields FROM files WHERE owner_uuid = ?`) or something like that.